test/JDBC/upgrade/latest/schedule

```diff
@@ -339,3 +339,4 @@ BABEL-CHECK-CONSTRAINT
 BABEL-3640
 sys_server_principals_dep
 sys_database_principals_dep
+app_name
```

Comment:
Add test to minor version update as well, 14.6 to latest

Reply:
In 14.6, the preparation script cannot work because `app_name()` is added in 14.7. mVU (Minor Version Upgrade) will be tested by https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/.github/workflows/minor-version-upgrade.yml